pallets/asset-manager/src/lib.rs

```diff
@@ -378,6 +415,31 @@ pub mod pallet {
 			});
 			Ok(())
 		}
+
+		/// Set min xcm fee for assets.
+		///
+		/// * `origin`: Caller of this extrinsic, the access control is specified by `ForceOrigin`.
+		/// * `asset_id`: AssetId to be haven min xcm fee.
+		/// * `min_xcm_fee`: Amount of min_xcm_fee.
+		#[pallet::weight(T::WeightInfo::set_min_xcm_fee())]
+		#[transactional]
+		pub fn set_min_xcm_fee(
+			origin: OriginFor<T>,
+			multicaltion: <T::AssetConfig as AssetConfig<T>>::AssetLocation,
+			#[pallet::compact] min_xcm_fee: u128,
+		) -> DispatchResult {
+			T::ModifierOrigin::ensure_origin(origin)?;
+			ensure!(
+				LocationAssetId::<T>::contains_key(&multicaltion),
+				Error::<T>::UpdateNonExistAsset
+			);
+			MinXcmFee::<T>::insert(&multicaltion, &min_xcm_fee);
```

Comment:
No, you don't want to check the `LocationAssetId` . The `MInXcmFee` map is for reserve locations, not the actual asset locations. For example Karura's 3 assets will have the same Reserve `(1, X1(Parachain(karura_id)))` That's the location that will be inserted, so the map should just be called  `MinXcmFeeOfReserves`

Reply:
Are you sure we need to add a new storage or associated type to `Config`

Reply:
Improve it in next release.
https://github.com/Manta-Network/Manta/issues/568